### PR TITLE
Added basic websocket proxyfing

### PR DESCRIPTION
**Pull request by phtrivier**

### CODE REVIEW

File: src/github.com/airvantage/moxy/server.go [hunk 1]
```diff
@@ -4,13 +4,14 @@ import (
...
 	"bytes"
 	"encoding/hex"
-	"git.eclipse.org/gitroot/paho/org.eclipse.paho.mqtt.golang.git/packets"
 	"io"
 	"io/ioutil"
 	"log"
 	"net"
 	"os"
 	"strconv"
+
```
Comment (jvermillard) [inline src/github.com/airvantage/moxy/server.go:13]: go fmt?

File: src/moxy-dummyauth/main.go [hunk 2]
```diff
@@ -85,3 +88,29 @@ func server(c net.Conn) {
 	}
 
 }
+
+// Read connection info from MQTT_URL env variable (eg "localhost:1884").
+// Defaults to "iot.eclipse.org:1883" is variable is not set.
+func mqttConnectionInfo() (string, int) {
+	host, port := "iot.eclipse.org", 1883
+
+	mqttURL := os.Getenv("MQTT_URL")
```
Comment (jvermillard) [inline src/moxy-dummyauth/main.go:97]: would be nice to avoid looking at env var for every MQTT connection

**Comment by jvermillard** [comment]
rebased onto the master

File: src/github.com/airvantage/moxy/server.go (hunk 1 above)
Comment (phtrivier) [inline src/github.com/airvantage/moxy/server.go:13]: Nope, atom's go mode uses [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) ; I'll change it back to `gofmt`.

**Pull request closed without merging**